plugin/src/main/scala/migrate/Messages.scala

```diff
@@ -98,6 +100,40 @@ object Messages {
          |
          |""".stripMargin
 
+  def migrateLibsStarting(projectId: String): String =
+    s"""|
+        |
+        |Starting to migrate libDependencies for $projectId
+        |""".stripMargin
+
+  def notMigratedLibs(libs: Seq[Lib]): String =
+    s"""|
+        |
+        |The following list of libs cannot be migrated.
+        |Please check the migration guide for more information. 
+        |${libs.map(_.toString).mkString("\n")}
+        |
+        |
+        |
```

Comment:
Do we need so many white lines? The output on the PR description looks a bit odd